examples/goproxy-socks/socksproxy.go

```diff
@@ -0,0 +1,73 @@
+// This example demonstrates how to configure goproxy to act as an HTTP/HTTPS proxy
+// that forwards all traffic through a SOCKS5 proxy.
+// The goproxy server acts as an aggregator, handling incoming HTTP and HTTPS requests
+// and routing them via the SOCKS5 proxy.
+// Example usage:
+// socks proxy with no auth:
+// 		go run socksproxy.go -v -addr ":8080" -socks "localhost:1080"
+// socks with auth:
+// 		go run socksproxy.go -v -addr ":8080" -socks "localhost:1080" -user "bob" -pass "123"
```

Comment:
Can you move all this part into a README file like all the other examples, maybe adding some more explaination that this is based on cascadeproxy, etc.?